README.md

```diff
@@ -140,6 +140,16 @@ kubectl create secret docker-registry flux-enterprise-auth \
   --docker-password=$ENTERPRISE_TOKEN
 ```
 
+### Migration of a bootstrap cluster
+
+To migrate a cluster that was bootstrapped, after the flux-operator is installed
+and the `FluxInstance` resource is created, the following steps are required:
+
+1. Checkout the branch of the Flux repository that was used to bootstrap the cluster.
+2. Replace the contents of the `flux-system/gok-components.yaml` with the `FluxInstance` YAML manifest.
```

Comment:
does this mean that we have to keep updating the Flux repository when updating the operator, i.e. changes to the `FluxInstance spec`? Couldn't we delete the `flux kustomization` and let the operator handle updates?

Reply:
Sure you could have the `FluxInstance` in TF state, or applied by OperatorHub or by any other tool/marketplace automation. We'll need extensive docs for how to do this depending on platform.

Reply:
> Couldn't we delete the flux kustomization and let the operator handle updates?

I guess you're refering to gotk-sync.yaml? If so, we can't delete it, that syncs all Flux CRs, we can only delete the gotk-components.yaml.

Reply:
I think we should have that as a follow-up. This would be a really neat transition.